### PR TITLE
client: workflow crud methods

### PR DESCRIPTION
**Pull request by danielnorberg**

### CODE REVIEW

File: styx-cli/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java [hunk 1]
```diff
@@ -20,19 +20,30 @@
...
+import com.spotify.apollo.Status;
+import com.spotify.styx.api.Api;
+import com.spotify.styx.model.Schedule;
+import com.spotify.styx.model.Workflow;
+import com.spotify.styx.model.WorkflowConfiguration;
+import com.spotify.styx.serialization.Json;
 import java.util.concurrent.CompletableFuture;
 import junitparams.JUnitParamsRunner;
 import junitparams.Parameters;
+import okhttp3.HttpUrl;
```
Comment (bergman) [inline styx-cli/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java]: com.squareup.okhttp.HttpUrl

Comment (bergman) [reply styx-cli/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java]: Not sure what the difference is...

Comment (honnix) [reply styx-cli/src/test/java/com/spotify/styx/client/StyxApolloClientTest.java]: We have both okhttp and okhttp3 as dependency. :O

**Comment by honnix** [comment]
I don't see why we had to introduce okhttp3. I verified styx-cli still works using okhttp. Let's remove `okhttp3` for now?

**Comment by honnix** [comment]
https://github.com/spotify/styx/pull/302

**Comment by danielnorberg** [comment]
Having only one okhttp dep sgtm =)

**Comment by honnix** [comment]
👍

**Pull request merged**